text/0000-optional-jquery.md

```diff
@@ -0,0 +1,248 @@
+use `this.element` to refer to the component's root element, and use the new DOM interaction helpers instead of jQuery 
+events triggered through `this.$()`. 
+
+### Addon migration
+
+One of the biggest problems to easily opt-out of jQuery is that many addons still depend on it. Many of these usages 
+seem to be rather "accidental", in that the full power of jQuery is not really needed for the given task, and could 
+be fairly easily refactored to use only native DOM APIs.
+
+For this reason this RFC encourages addon authors to not use jQuery anymore and to refactor existing usages whenever 
+possible! This certainly does not apply categorically to all addons, e.g. those that wrap jQuery plugins as 
+components and as such cannot drop this dependency.
+
+#### ember-try
+
+`ember-try`, which is used to test addons in different scenarios with different dependencies, should provide some means
+to define scenarios without jQuery, based on the jQuery integration flag introduced in this RFC.
+
+Furthermore the Ember CLI blueprint for addons should be extended to include no-jQuery scenarios by default, to make
+sure addons don't cause errors when jQuery is not present.
+
+#### emberobserver.com
+
+It would be very helpful to have a clear indication on [emberobserver.com](https://emberobserver.com/) which 
+addons depend on jQuery and which not. This would benefit users as to know which addons they can use without 
+jQuery, but also serve as an incentive for authors to make their addons work without it.
+
+Given the jQuery integration flag introduced in this RFC, this paves the way to automatically detect addons that are 
+basically declaring their independence from jQuery by having this flag set to `false`  (in their own repository).
+
+## Drawbacks
+
+### Churn
+
+A vast amount of addons still depend on jQuery. While as far as this RFC is concerned no jQuery based APIs will be
+deprecated and the default will still be to include jQuery, addons are nevertheless encouraged to remove their 
+dependency on jQuery, which will add some considerable churn to the addon ecosystem. As of writing this, there are:
+* [475 addons](https://emberobserver.com/code-search?codeQuery=Ember.%24) using `Ember.$`
+* [479 addons](https://emberobserver.com/code-search?codeQuery=this.%24&fileFilter=addon%2Fcomponents) using `this.$` in
+components
+* [994 addons](https://emberobserver.com/code-search?codeQuery=this.%24&fileFilter=tests) using `this.$` in tests
+
+Among these are still some very essential addons like `ember-data`, which still relies on `$.ajax`, see 
+[#5320](https://github.com/emberjs/data/issues/5320).
+
+A good amount of that churn can be mitigated by having a codemod that migrates tests (see "Testing" above).
+
+## Alternatives
+
+Continue to depend on jQuery.
+
+## Unresolved questions
+
+### jquery.Event vs. native events
```

Comment:
In my opinion, this question needs to be somewhat resolved before this RFC can proceed, as it pretty much derails the whole show.

Reply:
Yep, I agree that we need to reach a consensus on the best path forward here.